packages/js/manifests/polywrap/src/formats/polywrap.docs/0.1.0.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export interface DocsManifest {
+  /**
+   * Polywrap Docs manifest format version.
+   */
+  format: "0.1.0";
+  /**
+   * A description of the wrap.
+   */
+  description?: string;
+  /**
+   * Path to the wrap's logo.
+   */
+  logo?: string;
+  /**
+   * Website related to the wrap.
+   */
+  websiteUrl?: string;
```

Comment:
NIT: I think we can do without the `Url` at the end. Other popular manifests (`Cargo.toml`, `package.json`) don't do this.

Also if we follow this naming convention, then we'd also have `logoPath` and `readmePath`. Better to just omit it IMO.

Reply:
Same for `repositoryUrl`